Makefile.am

```diff
@@ -16,6 +16,9 @@ dist_systemdunit_DATA = \
 systemdunit_DATA = \
 	dev-disk-by\\x2dlabel-eos\\x2dswap.swap \
 	var-endless\\x2dextra.mount
+
+udevrulesdir=/lib/udev/rules.d
```

Comment:
Unfortunately, this will break distcheck since it needs the install to all be under `$prefix`. I'll add a commit to handle that.